sound/soc/sof/intel/ext_manifest.h

```diff
@@ -21,7 +21,9 @@
 enum sof_cavs_config_elem_type {
 	SOF_EXT_MAN_CAVS_CONFIG_EMPTY		= 0,
 	SOF_EXT_MAN_CAVS_CONFIG_CAVS_LPRO	= 1,
-	SOF_EXT_MAN_CAVS_CONFIG_LAST_ELEM	= SOF_EXT_MAN_CAVS_CONFIG_CAVS_LPRO, /**< keep up to date with last enum */
+	SOF_EXT_MAN_CAVS_CONFIG_OUTBOX_SIZE,
+	SOF_EXT_MAN_CAVS_CONFIG_INBOX_SIZE,
+	SOF_EXT_MAN_CAVS_CONFIG_LAST_ELEM = SOF_EXT_MAN_CAVS_CONFIG_INBOX_SIZE, /**< keep up to date with last enum */
```

Comment:
This looks a bit odd to have explicit values for some members, but not all. It seems FW has CAVS_LPRO set to 1, and I think EMPTY was added later for other reasons. So maybe this ok, but this does pop up form the code.

Reply:
I can put all explicit number also. but I need to sync up with fw header also. Can I keep current one? I will add TODO in https://github.com/thesofproject/linux/issues/2491